### PR TITLE
feat: [Staging Table] staging table for raw ingest files

### PR DESCRIPTION
**Pull request by antonyip**

### CODE REVIEW

File: models/staging/stg_ant_ingest.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    select
+        ingest_timestamp::timestamp as ingest_timestamp,
+        try_parse_json(ingest_data) as parsed_data
+    from {{ source("ingest","src_old_ant_ingest") }}
+    where     
+        {% if is_incremental() %}
+            ingest_timestamp::timestamp > (select max(ingest_timestamp) from {{ this }})
+        {%- else -%}
+            true
+        {% endif %}
```
Comment (chuxinh) [inline models/staging/stg_ant_ingest.sql:23]: isn't it the same as the macro for incremental build?

File: models/staging/stg_ant_ingest.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        and ingest_timestamp < '2022-03-07 15:00:00.000'
```
Comment (chuxinh) [inline models/staging/stg_ant_ingest.sql:24]: what's the reason for this?

File: models/staging/stg_ant_ingest.sql (hunk 1 above)
Comment (chuxinh) [reply models/staging/stg_ant_ingest.sql:23]: actually how come you have old and new ingest data?

File: models/sources.yml [hunk 3]
```diff
@@ -47,3 +47,51 @@ sources:
...
+            description: Market Cap of the coin in USD.
+          - name: timestamp
+            description: When the data was ingested.
+      - name: ant_ingest
+        description: "Ingest from ant"
+        columns:
+          - name: ingest_timestamp
+            description: What time this infomations was ingested
+          - name: ingest_data
+            description: the json that was ingested
```
Comment (chuxinh) [inline models/sources.yml]:
```suggestion
            description: The json that was ingested.
```

**Review by chuxinh — APPROVED** [review]
Just some comments but LGTM

File: models/staging/stg_ant_ingest.sql (hunk 1 above)
Comment (antonyip) [reply models/staging/stg_ant_ingest.sql:23]: [Macro] There's a cast on ingest_timestamp. (Ingest Table is setup to be varchar[16mb])
Old Table = copied over the old data from harmony,
new Table the current ingest table. There's some overlap in the timing thus why its setup this way...

File: models/staging/stg_ant_ingest.sql (hunk 2 above)
Comment (antonyip) [reply models/staging/stg_ant_ingest.sql:24]: There's some overlap in the timing thus why its setup this way...

**Pull request merged**